packages/transformer/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -1085,13 +1085,22 @@ describe("IModelTransformer", () => {
     targetDb.close();
   });
 
-  /** gets a mapping of element ids to their invariant content */
+  /** gets a mapping of element ids to their content ignoring or removing variance that is expected when transforming */
   async function getAllElementsInvariants(db: IModelDb, filterPredicate?: (element: Element) => boolean) {
+    // The set of element Ids where the fed guid should be ignored (since it can change between transforms).
+    const ignoreFedGuidElementIds = new Set<Id64String>([
+      IModel.rootSubjectId,
+      IModel.dictionaryId,
+      "0xe",
```

Comment:
add a comment saying this is the reality data container (check, I forget what it is)

Reply:
done